### PR TITLE
Rename variables to placeholdersync

### PR DESCRIPTION
**Pull request by ihcomega56**
Rename variables in GH Actions scripts to `placeholdersync` based on the linting error.

## PR Checklist

- [x] I've read the [contributing guidelines](https://github.com/pyrsia/.github/blob/main/contributing.md).
- [x] I've read ["What is a Good PR?"](https://github.com/pyrsia/pyrsia/blob/main/docs/community/get_involved/good_pr.md)
- [x] I've included a good title and brief description along with how to review them.
- [x] I've linked any associated an [issue](https://github.com/pyrsia/pyrsia/issues).

### CODE REVIEW

File: .github/workflows/chartmuseum.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ cd ../..
 
 python3 $WORKSPACE/.github/workflows/genlistingsyncoptimized.py ${listing} gs://helmrepo/repos repos
 # Generate pretty directory listing web pages
-python3 $WORKSPACE/.github/workflows/genlisting.py -r -d
```
Comment (betarelease) [inline .github/workflows/chartmuseum.sh:25]: was this change intended?

Comment (ihcomega56) [reply .github/workflows/chartmuseum.sh:25]: Yes I updated the short name from `-d` to `-p` as it follows the change of long name.

**Review by fishseabowl — APPROVED** [review]

**Review by betarelease — APPROVED** [review]
